src/ILVerify/tests/TestDataLoader.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Internal.IL;
+using Internal.TypeSystem.Ecma;
+
+namespace ILVerify.Tests
+{
+    /// <summary>
+    /// Helper method to load the binaries generated based on the il code, which drive the tests
+    /// </summary>
+    public static class TestDataLoader
+    {
+        /// <summary>
+        /// The folder with the binaries which are compiled from the test driver IL Code
+        /// </summary>
+        public static string TESTASSEMBLYPATH = @"..\..\..\ILTests\";
+
+        public static EcmaModule GetModuleForTestAssembly(string assemblyName)
+        {
+            var _typeSystemContext = new SimpleTypeSystemContext();
+
+            var systemRuntime = System.Runtime.Loader.AssemblyLoadContext.Default.LoadFromAssemblyName(new System.Reflection.AssemblyName("System.Runtime"));
+            var systemPrivateCoreLib = System.Runtime.Loader.AssemblyLoadContext.Default.LoadFromAssemblyName(new System.Reflection.AssemblyName("System.Private.CoreLib"));
```

Comment:
It may be better to use `typeof(Object).Assembly` here. It will make the tests work on both CoreCLR where the system module is System.Private.CoreLib, and desktop where the system module is mscorlib.